src/Dfe.PlanTech.DatabaseUpgrader/Scripts/2024/20240522_1456_CreateDeleteEstablishmentDataSProc.sql

```diff
@@ -0,0 +1,17 @@
+CREATE OR ALTER PROCEDURE [dbo].[deleteDataForEstablishment] @establishmentRef nvarchar(50)
+AS
+    BEGIN TRY
+        BEGIN TRAN
+            DELETE R
+            FROM [dbo].[response] R
+            JOIN [dbo].[submission] S ON R.[submissionId] = S.[id]
+            WHERE [S].[establishmentId] = (SELECT [id] FROM [dbo].[establishment] WHERE establishmentRef = @establishmentRef)
```

Comment:
This needs to delete from the submission table too, I'd put the establishmentId var back so you can use for both

Reply:
but this is much better, nice one 🎉

Reply:
I need to stop coding whilst i'm tired.... Sorted?